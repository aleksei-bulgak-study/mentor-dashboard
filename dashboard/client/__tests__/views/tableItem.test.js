import React from 'react';
import { shallow } from 'enzyme';
import TableItem from '../../src/views/tableItem';

describe('TableItem', () => {
  test('should match snapshot', () => {
    const props = {
      color: 'green', mark: 95, link: 'https://example.com',
    };
    const wrapper = shallow(<TableItem {...props} />);
    expect(wrapper).toMatchSnapshot();
  });

  test('should match snapshot when link is missed', () => {
    const props = {
      color: 'green', mark: 95, link: 'https://example.com',
    };
    const wrapper = shallow(<TableItem {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
