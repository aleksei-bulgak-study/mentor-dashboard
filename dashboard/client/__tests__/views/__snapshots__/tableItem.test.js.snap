// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableItem should match snapshot 1`] = `
<td
  className="green"
  title={95}
>
  <a
    className="student-results__score"
    href="https://example.com"
    rel="noopener noreferrer"
    target="_blank"
  >
    95
  </a>
</td>
`;

exports[`TableItem should match snapshot when link is missed 1`] = `
<td
  className="green"
  title={95}
>
  <a
    className="student-results__score"
    href="https://example.com"
    rel="noopener noreferrer"
    target="_blank"
  >
    95
  </a>
</td>
`;
